.github/workflows/swift-ci.yml

```
@@ -0,0 +1,91 @@
+#
+# .github/workflows/swift-ci.yml
+#
+---
+name: Swift CI
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-swift:
+    name: Build Swift (macos-latest)
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Select Xcode
+        run: |
+          # Use the latest stable Xcode version
+          sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
```

Are we sure about `Xcode.app` being the latest stable?

```suggestion
          sudo xcode-select -s /Applications/Xcode.app
```